README.md

```diff
@@ -203,6 +204,16 @@ This command would generate the following configuration:
 }
 ```
 
+### Debugging rails applications
+
+Install `debug` gem. Verify by running `bundle exec rdbg -v`
+
+Make sure you're running your application with rdbg, e.g. `bundle exec rdbg -O -n -c -- bin/rails server -p 3000`
+
+You should be able to use vscode debugger attach configuration to connect to your running app
```

Comment:
Launching the attach task should display a list of existing sockets for your running applications. Select the one you'd like to connect to and start debugging.